app/views/rentals/_form.html.erb

```diff
@@ -1,64 +1,149 @@
-<%= form_for @rental, method: :post do |f| %>
-  <div class="form-group col-xs-12">
-    <%= f.label :type, 'Cart Type', class: 'control-label col-xs-2' %>
-    <div class="col-xs-6">
-      <%= f.select :item_type_id, options_from_collection_for_select(@item_types, "id", "name"), {}, class: "selectpicker form-control" %>
-    </div>
-  </div>
+<ul class="nav nav-tabs">
+  <li class="nav-item active">
+    <a class-"nav-link active" data-toggle="tab" href="#single_day">Single Day</a>
+  </li>
+  <li class="nav-item">
+    <a class-"nav-link" data-toggle="tab" href="#multiple_day">Multiple Day</a>
+  </li>
+</ul>
 
-  <div class="form-group col-xs-12">
-    <%= f.label :name, 'Cart Name', class: 'control-label col-xs-2' %>
-    <div class="col-xs-6">
-      <%= f.select :item_id, options_from_collection_for_select(@items.where(item_type_id: params["item_type_id"], deleted_at: nil).order(name: :asc), "id", "name"), {}, class: "selectpicker form-control" %>
-    </div>
-  </div>
+<div class="tab-content">
+  </br>
+  <div class="tab-pane active" id="single_day" role="tabpanel">
+    <%= form_for @rental, method: :post do |f| %>
 
-  <div class="form-group col-xs-12">
-    <%= f.label :user, 'User', class: 'control-label col-xs-2' %>
-    <div class="col-xs-6">
-      <%= f.select :user_id, options_from_collection_for_select(@users.order(last_name: :asc), "id", "full_name"), {}, class: "selectpicker form-control", onchange: "update_disclaimer(this)" %>
-    </div>
-  </div>
+      <div class="form-group col-xs-12">
+        <%= f.label :type, 'Cart Type', class: 'control-label col-xs-2' %>
+        <div class="col-xs-6">
+          <%= f.select :item_type_id, options_from_collection_for_select(@item_types, "id", "name"), {}, class: "selectpicker form-control" %>
+        </div>
+      </div>
 
-  <div class="form-group col-xs-12">
-    <%= f.label :start_time, 'Start Date', class: 'control-label col-xs-2' %>
-    <div class="col-xs-6">
-      <%= f.date_field :start_time, value: @start_date, required: true, class: "form-control" %>
-    </div>
-  </div>
+      <div class="form-group col-xs-12">
+        <%= f.label :name, 'Cart Name', class: 'control-label col-xs-2' %>
+        <div class="col-xs-6">
+          <%= f.select :item_id, options_from_collection_for_select(@items.where(item_type_id: params["item_type_id"], deleted_at: nil).order(name: :asc), "id", "name"), {}, class: "selectpicker form-control" %>
```

Comment:
I added a default scope in PR #109 and changed the code in items_controller but I overlooked this code, that's my fault sorry about that. 
@owlee @phantompatrol 

> https://github.com/Cobchise/probable-engine/blob/4e130c394adfea2ef4b0c45a026e28fa5d1dd296/app/models/item.rb